src/EnergyPlus/ThermalComfort.cc

```diff
@@ -2629,6 +2631,7 @@ namespace ThermalComfort {
         int j;
         bool weathersimulation;
         Real64 inavgdrybulb;
+        int const numHeaderRowsInEpw = 8;
```

Comment:
There's a method in WeatherManager for this: https://github.com/NREL/EnergyPlus/blob/575a350ad418265382f0dd3f0c06ce038f4f8636/src/EnergyPlus/WeatherManager.cc#L8325

Can we move to using that instead of hardcoding the header offset?